app/views/favorites/index.html.erb

```diff
@@ -1,7 +1,13 @@
+<% if @fav_pets.empty? %>
+  <h3>You have not favorited any pets.</h3>
+<% else %>
+  <%= button_to 'Remove All Pets from Favorites', "/favorites", method: :delete, class: 'button' %>
+<% end %>
```

Comment:
This is for the display message, pretty straight forward. If there aren't any favorited pets, then the message will display. If there are favorited pets, then there will be a message to remove all favorites, which route to the destroy action. (from above, this is the only spot where the pet_id will equal nil)